net.sourceforge.Pdfedit.json

```diff
@@ -1,12 +1,13 @@
 {
     "app-id": "net.sourceforge.Pdfedit",
     "runtime": "org.freedesktop.Platform",
-    "runtime-version": "23.08",
+    "runtime-version": "24.08",
     "sdk": "org.freedesktop.Sdk",
     "command": "pdfedit",
     "finish-args": [
         "--share=ipc",
-        "--socket=x11",
+        "--socket=fallback-x11",
+        "--socket=wayland",
```

Comment:
did you even test it?

*hint* qt3 does NOT support wayland.

Reply:
> did you even test it?

Not locally, I was hoping for a success build on MR and then test the result, but not luck this time it seems Boost fails to build so yeah I'll need to build locally to debug the Boost fail as buildbot logs are insufficient.
 
> _hint_ qt3 does NOT support wayland.

Oh  yes an oversight on my part as I've been updating that lines in several of my flatpaks. Thanks for the heads up 👍🏻 .

I've also noticed that my Java app flatpaks still need to stay on X11, there's still no OpenJDK that supports wayland although it's been worked for future versions.